.github/workflows/build_and_test.yml

```diff
@@ -669,7 +669,7 @@ jobs:
             neondatabase/compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }}-${{ matrix.version.debian }}-${{ matrix.arch }}
 
       - name: Build neon extensions test image
-        if: matrix.version.pg == 'v16'
+        if: matrix.version.pg >= 'v16'
```

Comment:
Do we want to test both versions here? I think only the latest is enough.
Please, add a comment, if this was intentional

Reply:
With different sets of extensions on Postgres 16 and 17 (and different versions), I think it makes sense to run tests on both versions.

TIL: that we can compare stings in GitHub actions, looks neat!